server/services/chat/olly_chat_service.ts

```diff
@@ -114,12 +185,17 @@ export class OllyChatService implements ChatService {
       verbose: true,
     };
 
-    return await this.requestAgentRun(rootAgentId, parametersPayload, context);
+    return await this.requestAgentRun(parametersPayload);
   }
 
   abortAgentExecution(sessionId: string) {
     if (OllyChatService.abortControllers.has(sessionId)) {
       OllyChatService.abortControllers.get(sessionId)?.abort();
     }
   }
+
+  // only used for test
+  public resetRootAgentId() {
+    OllyChatService.rootAgentId = undefined;
+  }
```

Comment:
If only for test, maybe we can initialize a new instance of `OllyChatService` in beforeEach?

Reply:
`rootAgentId` is static, the value is used by all instances of `OllyChatService`.

Reply:
Oh, I missed that. Then not able to come up with better solution to get rid of this method.